srcpkgs/qomui/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'qomui'
+pkgname=qomui
+version=0.8.1
+revision=1
+build_style=python3-module
+hostmakedepends="python3 python3-setuptools"
+depends="
+	python python3 python3-PyQt5 python3-dbus python3-psutil python3-requests \
```

Comment:
don't skip the first line and use whitespace, not tabs, for identation on following lines.

also don't need to use `\` inside quoted variables